packages/curve-ui-kit/src/shared/ui/TradingSlider.tsx

```diff
@@ -38,6 +38,7 @@ export const TradingSlider = ({
     }}
   >
     <Slider
+      className="borderless"
```

Comment:
Maybe we can introduce some extra props, but className is fine. Can we use a constant though, so it's easy to find?